test/mx-8/theme/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Do we need the theme and widgets committed to this test folder?

Reply:
Yes, because otherwise the styling of the app will break due to a missing atlas UI dependency. The project also depends on other widgets, so the widgets folder needs to be included as well. However, the BootstrapTooltip widget isn't committed.